src/hotspot/cpu/riscv/stubGenerator_riscv.cpp

```diff
@@ -4277,6 +4277,129 @@ class StubGenerator: public StubCodeGenerator {
     return (address) start;
   }
 
+  // rotate vector register left with shift bits, 32-bit version
+  void rotate_left_imm(VectorRegister rv, uint32_t shift, VectorRegister tmp_vr) {
```

Comment:
This should be in `macroAssembler_riscv.cpp` instead. I would also call it something like `vrolwi`. That'll more closely match the [`vrol`](https://github.com/riscv/riscv-crypto/blob/c8ddeb7e64a3444dda0438316af1238aeed72041/doc/vector/insns/vrol.adoc#L5)